src/views/not_found.hbs

```diff
@@ -1 +1 @@
-<h1> Sorry we couldn't find any startup in our database that match your selections</h1>
+<h1> Sorry we could not find what you are searching for</h1>
```

Comment:
We have a navbar partial showing on all pages, and you can navigate to homepage from there